{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/common/views.py

```diff
@@ -1,10 +1,25 @@
+from django.contrib.staticfiles.storage import staticfiles_storage
+from django.http import FileResponse, Http404
 from django.shortcuts import render
 from django.template.exceptions import TemplateDoesNotExist
-from rest_framework import status
+from django.views.generic.base import View
+from rest_framework import permissions, status
 
 
 def index(request):
     try:
         return render(request, "index.html")
     except TemplateDoesNotExist:
         return render(request, "core/index-placeholder.html", status=status.HTTP_404_NOT_FOUND)
+
+
+class WellKnownExampleFile(View):
```

Comment:
FYI I dont think you need to create a whole view 
https://github.com/thinknimble/tn-spa-bootstrapper/blob/c41cfa9ab799833beb5650eea13c5a6dde42d2ce/%7B%7Bcookiecutter.project_slug%7D%7D/server/%7B%7Bcookiecutter.project_slug%7D%7D/common/favicon_urls.py#L15